x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -98,7 +127,51 @@ const navigationTree: NavigationTreeDefinition = {
   footer: [
     {
       type: 'navGroup',
-      ...getPresets('management'),
+      id: 'projest_settings_project_nav',
+      title: 'Project settings',
+      icon: 'gear',
+      defaultIsCollapsed: true,
+      breadcrumbStatus: 'hidden',
+      children: [
+        {
+          id: 'settings',
+          children: [
+            {
+              link: 'monitoring',
```

Comment:
Probably not the right spot for a comment, but we can link to the `/app/management` app view. This will eventually be replaced with a director overview of the management options available for the project type.

Reply:
good catch.  Fixed it here 

https://github.com/elastic/kibana/pull/160866/commits/499459d7ecfbe111dadf00283f6e0349fa84bc34


https://github.com/elastic/kibana/assets/3369346/00a76690-54a4-4929-ba6c-d0de579db5ed